core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -187,4 +187,48 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging with RackAwareT
     }
     checkReplicaDistribution(assignment, rackInfo, rackInfo.size, alteredNumPartitions, replicationFactor)
   }
+
+  @Test
+  def testDescribeAndListTopicsMarkedForDeletion() {
+    val brokers = List(0)
+    val topic = "testtopic"
+    val markedForDeletionDescribe = "MarkedForDeletion"
+    val markedForDeletionList = "marked for deletion"
+    TestUtils.createBrokersInZk(zkUtils, brokers)
```

Comment:
For `listTopics()` we could easily mock `zkUtils` as it only does 1 call. However for `describeTopics()`, `zkUtils` is used in many calls so I can't see the real benefit of mocking just one.

That said, we can still add a `isTopicMarkedForDeletion` helper method as `zkUtils.pathExists(getDeleteTopicPath(topic))` is used in a few other places too